README.md

```diff
@@ -37,6 +37,13 @@ to the RDO project.
 * Add your name and IRC nick to the [mentor list](#mentors)
 * Follow the steps in [I want to add an easy fix issue](#i-want-to-add-an-easyfix-issue)
 
+## Checklist for sending your first RDO gerrit Review:
+* [ ] Add the changed files using git add command
+* [ ] In the commit message: 
+      * First line: What you have fixed?
+      * Second Line: Fixed in <RDO easy Fix issue link>
```

Comment:
Technically this was fixed in the commit, no in the easy fix issue. Message can be something like "Fix for issue <RDO easy Fix issue link>"